src/codegen/codegen.utils.ts

```diff
@@ -109,3 +109,15 @@ export function mergeRedirects(recording: ProxyData[]) {
 
   return result
 }
+
+export const removeWebsocketRequests = (recording: ProxyData[]) => {
+  return recording.filter((data) => {
+    return getUpgradeHeader(data.request.headers) !== 'websocket'
+  })
+}
+
+export const recordingCleanup = (recording: ProxyData[]) => {
+  let cleanedRecording = mergeRedirects(recording)
```

Comment:
It's not cleaned up at this point yet, so this doesn't seem like the best name.
Maybe we can simply do the following:
```ts
removeWebsocketRequests(mergeRedirects(recording))
```
it's not a long term solution though, we may want to use function composition or find another way to keep it readable as the number of steps grows 🤔

Reply:
I wouldn't be in favor of chaining the calls because it will be less readable 🤔 

From my experience it was common to have the final name of the item you are going to return and overwrite it in multiple steps, it makes it readable what actions you are taking on it and the name reflects the value that get returned at the end of the function 🤔 
(technically even after the first step is cleaned because applied a "cleanup" step 😄 )

Reply:
What about something like this:
```ts
import { flow } from 'lodash'

// ...

const cleanRecording = flow(
  mergeRedirects,
  removeWebsocketRequests
)(recording)

return cleanRecording
```
?